src/test/java/org/sonar/plugins/github/PullRequestIssuePostJobTest.java

```diff
@@ -167,6 +173,99 @@ public void testPullRequestAnalysisWithNewIssues() {
   }
 
   @Test
+  public void testSortIssues() {
+    ArgumentCaptor<String> commentCaptor = forClass(String.class);
+
```

Comment:
Can't you extract construction of an issue in a method to avoid this duplication? Something like newMockedIssue("src/Foo.php", 8, Severity.BLOCKER, "msg1")